installer.sh.in

```diff
@@ -957,6 +982,18 @@ menu_network() {
     fi
 }
 
+validate_useraccount() {
```

Comment:
this function should also check if the username is valid (see useradd(1)) to prevent #256